.pre-commit-config.yaml

```diff
@@ -19,15 +19,16 @@ repos:
   - id: lint-frontend
     name: Linting of JS, Vue, SCSS and CSS files on the frontend
     description: This hook handles all frontend linting for Kolibri
-    entry: yarn run lint-frontend
+    entry: yarn run lint-frontend:format
     language: system
     files: ^kolibri_explore_plugin/assets/src/.*\.(js|vue|scss|css)$
   - id: lint-packages
-    name: Linting of JS, Vue, SCSS and CSS files on the packages
+    name: Linting of JS and Vue files on the packages
     description: This hook handles all pacakges linting
-    entry: yarn run lint-frontend
+    entry: yarn run lerna run lint
+    pass_filenames: false
     language: system
-    files: ^packages/.*\.(js|vue|scss|css)$
```

Comment:
Good point. I don't know why but the lint in template-ui or eos-components is not checking errors in the SCSS files.